creme/creme_core/views/entity.py

```diff
@@ -975,6 +975,8 @@ class EntitiesToMergeSelection(base.EntityRelatedMixin,
     mode = listview.SelectionMode.SINGLE
     entity1_id_arg = 'id1'
 
+    reload_url = reverse_lazy('creme_core__select_entity_for_merge')
```

Comment:
Generally we use a simple string with an attribute "foo_url_name' (see "bricks_reload_url_name").